### PR TITLE
Update supporter profile

### PR DESCRIPTION
**Pull request by arizephyr123**
# Description

For Supporter Profiles:
- Updated styling to reflect design files
- Added the Profile tab placeholder for future Saved Campaigns
- Created components/ComingSoon.js as filler
- Updated components/CampBlankScreen.js to change wording based on supporter/organization
 - Updated the Details Section
     - removed About Me and Species and Habitats
     - moved social icons to ConnectFurther.js

For Conservationist Profiles:
 - removed Big Issues, Hand, SupportOurMission and Donate button
 - moved social icons to ConnectFurther.js

In LiveFeed:
- Changed Campaign urgency header to not render if no urgency data
- Created TakeActionCta.js which currently just links to organizations donationLink url
- Styled Take Action button as per design files

## Checklist
- [ x ] I have performed a self-review of my own code
- [ x ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: components/Profile/ProfileBody.js [hunk 1]
```diff
@@ -13,17 +13,25 @@ export default class ProfileBody extends Component {
   constructor(props) {
     super(props);
 
-    let routes = this.props.profile.roles === 'supporter' ? [
-      { key: 'details', title: 'Details' }
-    ] : [
-      { key: 'campaigns', title: 'Campaigns' },
-      { key: 'details', title: 'Details' }
-    ]
```
Comment (mtourj) [inline components/Profile/ProfileBody.js]: The deleted version of this block is the correct version, the new version will cause a duplication of the location tab

File: components/Profile/ProfileBody.js [hunk 2]
```diff
@@ -60,9 +70,12 @@ export default class ProfileBody extends Component {
 
   renderScene = SceneMap({
     campaigns: () => <Campaigns profile={this.props.profile} />,
-    location: () => <Location profile={this.props.profile} />,
-    details: () => <Details profile={this.props.profile} />
-  })
+    //    location: () => <Location profile={this.props.profile} />,
+    location: () => <Details profile={this.props.profile} />,
```
Comment (mtourj) [inline components/Profile/ProfileBody.js]: location should point to location

File: components/FeedScreen/FeedCampaign.js [hunk 3]
```diff
@@ -24,6 +24,8 @@ import Ellipse from '../../assets/jsicons/Ellipse';
 import CommentIcon from '../../assets/jsicons/CommentIcon';
 import MapMarker from '../../assets/jsicons/headerIcons/map-marker';
 import CampaignActionSheet from '../Reports/CampaignActionSheet';
+import TakeActionCta from '../TakeAction/TakeActionCta';
+// components/TakeAction/TakeActionCta.js
```
Comment (RashaMou) [inline components/FeedScreen/FeedCampaign.js]: I'd recommend taking out unnecessary comments.

File: components/FeedScreen/FeedCampaign.js [hunk 4]
```diff
@@ -526,7 +528,38 @@ const FeedCampaign = props => {
             />
           </TouchableOpacity>
         </View>
-        {/* <Text style={styles.timeText}>{timeDiff}</Text> */}
+        {/* HERE !! */}
+        <TakeActionCta profile={props.currentUserProfile} />
+        {/* <View style={styles.campMission}>
```
Comment (RashaMou) [inline components/FeedScreen/FeedCampaign.js]: If we don't need this, we can remove

**Pull request merged**